tutorial.md

```diff
@@ -33,31 +33,17 @@ This section will walk you through installing the following components:
 
 ### runc
 
-Download the `runc` release binary:
+CRI-O requires the version of runc below or newer.
 
-```
-wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc2/runc-linux-amd64
-```
-
-Set the executable bit and copy the `runc` binary into your PATH:
-
-```
-chmod +x runc-linux-amd64
-```
-
-```
-sudo mv runc-linux-amd64 /usr/bin/runc
-```
-
-Print the `runc` version:
+You can check your `runc` version:
 
 ```
 runc -version
 ```
 ```
-runc version 1.0.0-rc2
-commit: c91b5bea4830a57eac7882d7455d59518cdf70ec
-spec: 1.0.0-rc2-dev
+runc version 1.0.0-rc3
+commit: 75f8da7c889acc4509a0cf6f0d3a8f9584778375
+spec: 1.0.0-rc5
```

Comment:
@mrunalp mentioned to have a new RC6 cut to have 639454475cb9c8b861cc599f8bcd5c8c790ae402 in it, otherwise this runc version is still not fully compatible